apps/src/templates/projects/PublicGallery.jsx

```diff
@@ -42,18 +42,25 @@ class PublicGallery extends Component {
    * See the PropTypes of each component for a definition of each format.
    */
   mapProjectData(projectLists) {
-    return _.mapValues(projectLists, projectList => {
-      return projectList.map(projectData => {
-        return {
+    let allFeaturedProjects = [];
+    for (const projectListName in projectLists) {
+      if (projectListName === 'library') {
```

Comment:
We're returning libraries from the server when we request featured projects? Do we actually feature libraries anywhere?

Reply:
Good point! I don't believe so. I think published libraries are fetch by the call to `fetch_section_libraries` since only libraries within a user's same section can be shared.
I'll go ahead and add this condition to the backend in `fetch_featured_published_projects`.